common_module.py

```diff
@@ -96,23 +150,22 @@ def handle_exception(exc_type, exc_value, exc_traceback):
                   help="Rotate matrices' positioning by 90 degrees")
 parser.add_option("-v", "--verbose", dest="verbose", action="count",
                   help="Increase verbosity (specify multiple times for more)")
-parser.add_option("-W", "--writemethod", dest="write_method", action="store", default="txtyaml",
-                  choices=['txtyaml', 'json'],
-                  help="Set the method used for dumping information, json, yaml, or native python (yaml format)")
+# TODO: dumping to intermediate yaml files has been deprecated.
+# It is now possible, instead, to dump an all-including python structure into a json file (the "document")
```

Comment:
see line 1251 in qtop.py. Briefly, a named tuple is created, combining the data from all scheduler files, and then saved in json format.